README.md

```diff
@@ -0,0 +1,53 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
+
+> This is a project for microverse 2.0 training
+
+![screenshot](./app_screenshot.png)
```

Comment:
- [ ] Please fix the screenshot in your README file.
README's showing pictures of the project help making your repo look professional and inviting for the viewers.